src/m365/aad/commands/o365group/o365group-recyclebinitem-restore.ts

```diff
@@ -13,7 +15,9 @@ interface CommandArgs {
 }
 
 interface Options extends GlobalOptions {
-  id: string;
+  id?: string;
```

Comment:
Since all these options are optional now, we should track their usage in the telemetry

Reply:
Well spotted, totally forgot about that 😃